### PR TITLE
New sormas version 1.62.0

### PR DESCRIPTION
**Pull request by robbmue**

### CODE REVIEW

**Review by fhauptmann — COMMENTED** [review]
Left redis stuff intact

**Comment by robbmue** [comment]
@JonasCir @fhauptmann  shouldn't the redis stuff be removed?

**Review by JonasCir — CHANGES_REQUESTED** [review]

File: .env [hunk 1]
```diff
@@ -2,8 +2,8 @@ SORMAS_POSTGRES_PASSWORD=password
 
 SORMAS_POSTGRES_USER=sormas_user
 
-SORMAS_DOCKER_VERSION=2.22.0-rc3
-SORMAS_VERSION=1.61.2
+SORMAS_DOCKER_VERSION=2.23.0-rc1
+SORMAS_VERSION=1.62.0
```
[the remark this answers is not in the document]
Comment (robbmue) [reply .env:6]: yeah, but to keep the process i wanted to release 1.62.0 first

File: apache2/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FROM httpd:2.4-alpine
 ARG SORMAS_SERVER_URL
 
-ARG SORMAS_VERSION=1.61.2
+ARG SORMAS_VERSION=1.62.0
```
[the remark this answers is not in the document]
Comment (robbmue) [reply apache2/Dockerfile:4]: so i can just skip 1.62.0?

File: .env [hunk 3]
```diff
@@ -93,6 +93,8 @@ CUSTOMBRANDING_LOGINBACKGROUND_PATH=
 # SORMAS Central
```
[the remark this answers is not in the document]
Comment (JonasCir) [reply .env:93]: fixed

File: sormas/start-server.sh [hunk 4]
```diff
@@ -278,6 +280,7 @@ fi
 echo "SORMAS2SORMAS"
 if [ ! -z  "$SORMAS2SORMAS_ENABLED" ]; then
   echo "SORMAS2SORMAS enabled"
+  sed -i -E "s~#?sormas2sormas.path=.*~sormas2sormas.path=${SORMAS2SORMAS_PATH}~" "${PROPERTIES_FILE}"
   sed -i -E "s/#?sormas2sormas.id=.*/sormas2sormas.id=${SORMAS2SORMAS_ID}/" "${PROPERTIES_FILE}"
   sed -i -E "s/#?sormas2sormas.keystoreName=.*/sormas2sormas.keystoreName=${SORMAS2SORMAS_KEYSTORENAME}/" "${PROPERTIES_FILE}"
   sed -i -E "s/#?sormas2sormas.keystorePass=.*/sormas2sormas.keystorePass=${SORMAS2SORMAS_KEYPASSWORD}/" "${PROPERTIES_FILE}"
```
[the remark this answers is not in the document]
Comment (JonasCir) [reply sormas/start-server.sh:281]: fixed

File: .env [hunk 5]
```diff
@@ -93,6 +93,8 @@ CUSTOMBRANDING_LOGINBACKGROUND_PATH=
...
 #SORMAS_CENTRAL_ENABLED=1
 CENTRAL_OIDC_URL=localhost:8080
+CENTRAL_ETCD_HOST=localhost:6379
+CENTRAL_ETCD_CA_PATH=/opt/sormas/central/etcd/ca.pem
 CENTRAL_REDIS_HOST=localhost:6379
```
[the remark this answers is not in the document]
Comment (JonasCir) [reply .env]: done

File: .env [hunk 6]
```diff
@@ -93,6 +93,8 @@ CUSTOMBRANDING_LOGINBACKGROUND_PATH=
 # SORMAS Central
 #SORMAS_CENTRAL_ENABLED=1
 CENTRAL_OIDC_URL=localhost:8080
+CENTRAL_ETCD_HOST=localhost:6379
```
[the remark this answers is not in the document]
Comment (JonasCir) [reply .env]: fixed

**Pull request merged**